### PR TITLE
feat(useInterval): support set the increase step for every interval

### PR DESCRIPTION
**Pull request by shuperry**
useInterval function can increase faster and decrease now by setting increase step.

### CODE REVIEW

File: packages/shared/useInterval/index.ts [hunk 1]
```diff
@@ -42,11 +49,12 @@ export function useInterval(interval: MaybeRefOrGetter<number> = 1000, options:
   const {
     controls: exposeControls = false,
     immediate = true,
+    step,
```
Comment (antfu) [inline packages/shared/useInterval/index.ts:52]:
```suggestion
    step = 1,
```

File: packages/shared/useInterval/index.ts [hunk 2]
```diff
@@ -42,11 +49,12 @@ export function useInterval(interval: MaybeRefOrGetter<number> = 1000, options:
...
     callback,
   } = options
 
   const counter = ref(0)
-  const update = () => counter.value += 1
+  const update = () => counter.value += (step || 1)
```
Comment (antfu) [inline packages/shared/useInterval/index.ts:57]:
```suggestion
  const update = () => counter.value += step
```

**Comment by shuperry** [comment]
@antfu Thanks for reviewing this PR, your suggestion is done.

**Comment by antfu** [comment]
I wonder when this would be useful? The `count` should means how many times the interval has been called. If you want count double, won't `const myCount = computed(() => count.value * 2)` serve your need already?

**Comment by shuperry-antler** [comment]
> I wonder when this would be useful? The `count` should means how many times the interval has been called. If you want count double, won't `const myCount = computed(() => count.value * 2)` serve your need already?

Thanks for the reply, I joined a step param because I need to countdown, not always increase though.

**Comment by antfu** [comment]
Oh you are mutating the counter? It shouldn't be designed usage tho

**Comment by shuperry-antler** [comment]
It Seems that computed can also serve the need, this pr can close now.

**Pull request closed without merging**